### ISSUE TITLE
Recent commit(s) break the plugin

### ISSUE THREAD

**Issue by chrisgrieser** [Wansmer/symbol-usage.nvim#48]
I just updated my plugins and it appears that one of the recent updates broke symbol-usage for me. This is the error which I think pops up any time I symbol refresh is requested (running commit `eb5780d`):

```
  Error  Error executing vim.schedule lua callback: .../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:185: attempt to index field 'methods' (a string value)
stack traceback:
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:185: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:198: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:198: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:198: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:198: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:198: in function '_walk'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:221: in function 'traversal'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:81: in function 'handler'
	...l/Cellar/neovim/0.9.5/share/nvim/runtime/lua/vim/lsp.lua:1393: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>
```

I reverted back to the previous commit `4c79eff` and there are no issues anymore.

**Comment by Wansmer** [comment]
#49 should fix this bug. Please, check it.

**Comment by chrisgrieser** [comment]
Can confirm, issue does not appear anymore.

Thanks for the quick fix!

**Comment by chrisgrieser** [comment]
Hold on, seems that now a different issue is occurring:

```
Error executing vim.schedule lua callback: .../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:231: attempt to perform arithmetic on a nil value
stack traceback:
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:231: in function 'traversal'
	.../nvim/lazy/symbol-usage.nvim/lua/symbol-usage/worker.lua:81: in function 'handler'
	...w/Cellar/neovim/0.9.5/share/nvim/runtime/lua/vim/lsp.lua:1393: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by Wansmer** [comment]
Can you provide an example of the code where you get this error?

**Comment by Wansmer** [comment]
So, I reproduced a bug. Fixed in #50

**Comment by chrisgrieser** [comment]
thanks, this fixed it
